docker/external/config-e2e-pypi-repo.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash -x
```

Comment:
`#!/bin/bash -x` -->

```bash
#!/usr/bin/env bash
set -e

if [[ ! -z ${DEBUG} ]]; then
    set -x
fi
```

- `set -e` : if the script meets some error, the script will exit.
- `set -x`: print verbose info

Reply:
Other bash scripts have the same issue.

Reply:
fixed